config/jobs/cert-manager/cert-manager/cert-manager-periodics.yaml

```diff
@@ -1,7 +1,11 @@
+# THIS FILE HAS BEEN AUTOMATICALLY GENERATED
+# Don't manually edit it; instead edit the "cmrel" tool which generated it
+# Generated with: /home/ashley/workspace/cert-manager-release/bin/cmrel generate-tests --mode current
 periodics:
   - agent: kubernetes
     annotations:
       description: Runs unit and integration tests and verification scripts
+      testgrid-alert-email: cert-manager-dev-alerts@googlegroups.com
```

Comment:
note: This email annotation is a recurring change. I think wherever we specify a testgrid dashboard we should specify this email too.